.github/workflows/deploy.yml

```diff
@@ -0,0 +1,25 @@
+name: Deploy
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
I'd add `if: github.repository == 'sgrif/diesel.rs-website'` here so that this won't run on the fork repos.